libevm/VM.cpp

```diff
@@ -214,19 +214,24 @@ void VM::interpretCases()
 		
 		CASE(CREATE)
 		{
+			if (m_schedule->haveStaticCall && m_ext->staticCall)
```

Comment:
I agree. Keep only `m_ext->staticCall` check.
You can add before `if`: `assert(!staticCall || haveStaticCall)` which checks implication `staticCall => haveStaticCall`.